.coveragerc

```diff
@@ -15,5 +15,4 @@
 [run]
 source=shotgun_api3
 omit=
-    shotgun_api3/lib/httplib2/*
-    shotgun_api3/lib/six.py
+    shotgun_api3/lib/*
```

Comment:
I am not sure about that. There are files in this folder that belong to us:

- sgsix.py
- sgtimezone.py
- mockgun

Reply:
Good catch